spec/client_spec.js

```diff
@@ -226,17 +226,35 @@ describe('Client', function() {
     });
 
     describe('#get', function() {
+      var requestsCount = 0;
+
       beforeEach(function() {
-        sandbox.spy(window, 'setTimeout');
+        requestsCount++;
       });
 
       it('returns a promise', function() {
-        expect(subject.get('ticket.subject')).to.be.a.promise;
+        var promise = subject.get('ticket.subject');
+
+        promise.catch(function() {});
+        expect(promise).to.be.a.promise;
```

Comment:
let's put this above the `catch`, that way if it isn't a promise we get a nice assertion failure message rather than a `called catch on undefined` error